R/install.R

```diff
@@ -792,3 +796,10 @@ renv_install_remotes_update <- function(records, project) {
   records
 
 }
+
+renv_install_report <- function(records) {
+  renv_pretty_print_records(
+    records,
+    preamble = "The following package(s) will be installed into the library"
```

Comment:
```suggestion
    preamble = "The following package(s) will be installed into the library:"
```

Reply:
Might also be a good opportunity to move from "package(s)" to just checking the number of packages to be installed?